src/layouts/LoginRegisterLayout.vue

```diff
@@ -37,12 +38,41 @@ if (window.Cypress) {
 
 export default defineComponent({
   name: 'LoginLayout',
+  setup() {
+    const imageMask = `url(${new URL('../assets/svg/image-mask.svg', import.meta.url).href})`;
+    const imageUrl = rideToWorkByBikeConfig.urlLoginRegisterBackgroundImage;
+
+    return {
+      imageMask,
+      imageUrl,
+    };
+  },
 });
 </script>
 
 <template>
-  <q-layout view="hHh lpR fFf">
-    <q-page-container>
+  <q-layout class="bg-primary" view="hHh lpR fFf">
+    <!-- Background image -->
+    <div
+      v-if="$q.screen.gt.md"
```

Comment:
`$q.screen.gt.md` return `true` if screen width is >= 1440 px, is this condition correct?

My tested configuration is monitor with 1366x768 px resolution, and background image is not rendered due screen width condition.

Reply:
And e2e tests doesn't catch this case.

Reply:
Yes, this was the setup. But, as you write, it seems more appropriate to start the "desktop" layout on 
1024px breakpoint and above.
I updated the condition to `v-if="$q.screen.gt.sm"` and edited the column layout for register and login page accordingly.
Added an E2E image test for different resolutions.

Reply:
Looks good to me.